src/DIRAC/MonitoringSystem/Client/DataOperationSender.py

```diff
@@ -0,0 +1,53 @@
+import DIRAC
+from DIRAC import S_OK, S_ERROR, gLogger
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+from DIRAC.AccountingSystem.Client.DataStoreClient import gDataStoreClient
+from DIRAC.AccountingSystem.Client.Types.DataOperation import DataOperation
+from DIRAC.MonitoringSystem.Client.MonitoringReporter import MonitoringReporter
+
+
+def sendData(baseDict, commitFlag=False, delayedCommit=False, startTime=False, endTime=False):
+    """
+    Sends the input to Monitoring or Acconting based on the monitoringOption
+    :param dict baseDict: contains a key/value pair
+    :param bool commitFlag: decides whether to commit the record or not.
+    :param bool delayedCommit: decides whether to commit the record with delay (only for sending to Accounting)
+    """
+
+    monitoringOption = Operations().getValue("DataManagement/MonitoringBackends", ["Accounting"])
+    if "Monitoring" in monitoringOption:
+        baseDict["ExecutionSite"] = DIRAC.siteName()
+        dataOperationReporter = MonitoringReporter(monitoringType="DataOperation")
+        dataOperationReporter.addRecord(baseDict)
+        if commitFlag:
+            result = dataOperationReporter.commit()
+            gLogger.verbose("Committing data operation to monitoring")
+            if not result["OK"]:
+                gLogger.error("Couldn't commit data operation to monitoring", result["Message"])
+                return result
+            gLogger.verbose("Done committing to monitoring")
+        return S_OK()
```

Comment:
I mean, if it's `Accounting` on the backend list, it wouldn't  even execute that point of the code or? Since that implies that `Monitoring` is the value gotten from `monitoringOption`, or?

Reply:
You fetch it as a list so I understand that it can be `["Accounting", "Monitoring"]` to have the two.